File: src/main/resources/static/app/system/entities/TelegramSettings.js
class TelegramSettings {

	constructor(token, chatId, username) {
		this.token = token;
		this.chatId = chatId;
		this.username = username;
    }
    
    getToken() {
		return this.token;
	}
	
	getChatId() {
		return this.chatId;
	}
	
	getUsername() {
		return this.username;
	}
	
	
}

File: src/main/java/com/iotassistant/models/TelegramSettings.java
package com.iotassistant.models;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class TelegramSettings {
	
	@Id
    @GeneratedValue
    private int id;
	
	private String token;
	
	private String chatId;
	
	private String username;

	public TelegramSettings() {
		super();
	}

	public TelegramSettings(String token, String chatId, String username) {
		super();
		this.token = token;
		this.chatId = chatId;
		this.username = username;
	}

	public String getToken() {
		return token;
	}

	public String getChatId() {
		return chatId;
	}

	public String getUsername() {
		return username;
	}
	
	

}
